Knock.net/RequestUtilities.cs

```diff
@@ -126,6 +126,14 @@ private static string UrlEncodeAndClean(string value)
                         result.Add(new KeyValuePair<string, string>(key, s));
                         break;
 
+                    case int i:
+                        result.Add(new KeyValuePair<string, string>(key, i.ToString()));
```

Comment:
Found out passing options like `page_size: 1` was not working unless using a string instead of a number.
Decided to fix it.